prime-router/src/main/kotlin/azure/LookupTableFunctions.kt

```diff
@@ -50,26 +41,30 @@ class LookupTableFunctions(
             route = "lookuptables/list"
         ) request: HttpRequestMessage<String?>
     ): HttpResponseMessage {
-        return getOktaAuthenticator().checkAccess(request) {
-            try {
-                val showInactive = request.queryParameters[showInactiveParamName]
-                    ?.equals("true", true) ?: false
-                // Return everything if showInactive is true
-                val list = lookupTableAccess.fetchTableList(showInactive)
-                when (request.httpMethod) {
-                    HttpMethod.HEAD -> {
-                        // Get the latest date of all the tables
-                        val lastModified = list.maxWithOrNull(compareBy { it.createdAt })?.createdAt
-                        HttpUtilities.okResponse(request, lastModified)
-                    }
-                    else -> {
-                        HttpUtilities.okResponse(request, mapper.writeValueAsString(list))
-                    }
+        // Do authentication
+        val claims = AuthenticatedClaims.authenticate(request)
+            ?: return HttpUtilities.unauthorizedResponse(request, authenticationFailure)
+        // No authorization check!   Anyone authenticated in ReportStream is allowed to access this function.
```

Comment:
This is typical of the authorization changes I made for LookupTable API. AuthenticatedClaims.authenticate tries both okta and server2server auth.

Then, for reads, do no authorization at all!   Everybody authenticated gets to read.

Reply:
I like the comment `// No authorization check` since if this code ever becomes copy pasta, then it's clear there's no auth.